src/examples/zkapps/voting/off_chain_storage.ts

```diff
@@ -0,0 +1,36 @@
+// Merkle Tree and off chain storage
```

Comment:
sorry to be annoying, but can we add this in a separate PR (and include at least a few basic tests) -- maybe a test contract?

Reply:
It was only meant to be a small wrapper for the off-chain storage so we dont have to work with two separate APIs (map and merkle tree) - but if you want I can put that into its own PR + tests and we might even add it to the Experimental namespace?

Reply:
ah okay, yeah this is fine for now